apis/appmesh/v1beta2/virtualgateway_types.go

```diff
@@ -322,6 +322,10 @@ type VirtualGatewaySpec struct {
 	// 	if absent, it selects no pod.
 	// +optional
 	PodSelector *metav1.LabelSelector `json:"podSelector,omitempty"`
+	// GatewayRouteSelector selects GatewayRoutes using labels to designate GatewayRout membership.
+	// This field follows standard label selector semantics; if present but empty, it selects all in that namespace.
```

Comment:
we need to mention that `if it's unspecified or present but empty, it selects all gatewayRoute in that namespace`